.github/workflows/ci_testing.yml

```diff
@@ -38,16 +38,9 @@ jobs:
         run: |
           mkdir -p build
           cd build
-          cmake -DCMAKE_BUILD_TYPE=RELEASE -DCOVERAGE=ON ..
+          cmake -DCMAKE_BUILD_TYPE=RELEASE -DUSGSCSM_EXTERNAL_DEPS=ON ..
           cmake --build .
       - name: Test Package
         run: |
           cd build
           ctest -VV
-      - name: Upload Coverage
-        uses: codecov/codecov-action@v1
-        with:
-          token: ${{ secrets.CODECOV_TOKEN }}
-          fail_ci_if_error: true
-          verbose: true
-        if: matrix.os == 'ubuntu-latest' && matrix.python-version == '3.9'
```

Comment:
Why are you getting rid of this? Was is faulty or is it unnecessary?

Reply:
We don't do code coverage on USGSCSM right now. I had copied this over from ALE which does do code coverage.